ga4gh/protocol.py

```diff
@@ -5,14 +5,46 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import sys
+import datetime
 import json
 import inspect
-import datetime
-import itertools
-from cStringIO import StringIO
 
-import avro.io
+import google.protobuf.json_format as json_format
```

Comment:
Oops, sorry. Looks like json_format is a module not a function. Ignore this comment.